lisa/sut_orchestrator/hyperv/platform_.py

```diff
@@ -0,0 +1,317 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from pathlib import PureWindowsPath
+from typing import Any, List, Optional, Type, cast
+
+from lisa import feature, schema, search_space
+from lisa.environment import Environment
+from lisa.node import RemoteNode
+from lisa.platform_ import Platform
+from lisa.tools import Cp, HyperV, Mkdir, PowerShell
+from lisa.util import LisaException
+from lisa.util.logger import Logger, get_logger
+from lisa.util.subclasses import Factory
+
+from .. import HYPERV
+from .context import get_node_context
+from .schema import HypervNodeSchema, HypervPlatformSchema
+from .serial_console import SerialConsole, SerialConsoleLogger
+from .source import Source
+
+
+class _HostCapabilities:
+    def __init__(self) -> None:
+        self.core_count = 0
+        self.allocable_memory_mib = 0
+
+
+class HypervPlatform(Platform):
+    @classmethod
+    def type_name(cls) -> str:
+        return HYPERV
+
+    @classmethod
+    def supported_features(cls) -> List[Type[feature.Feature]]:
+        return [SerialConsole]
+
+    def _initialize(self, *args: Any, **kwargs: Any) -> None:
+        self._hyperv_runbook = self._get_hyperv_runbook()
+        self._server = self._initialize_server_node()
+        self._host_capabilities = self._get_host_capabilities(self._log)
+        self._source_vhd: Optional[PureWindowsPath] = None
+        self._source_factory = Factory[Source](Source)
+        self._artifact_paths: Optional[List[PureWindowsPath]] = None
```

Comment:
call it `_source_files`, so it's consistent with other places.

Reply:
done.